### PR TITLE
KEP-1672: update docs for EndpointSliceTerminatingCondition

### PR DESCRIPTION
**Pull request by andrewsykim**
Update docs for EndpointSliceTerminatingCondition

https://github.com/kubernetes/enhancements/issues/1672

### CODE REVIEW

File: content/en/docs/concepts/services-networking/endpoint-slices.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ Services will always have the `ready` condition set to `true`.
 
 #### Serving
 
-{{< feature-state for_k8s_version="v1.22" state="beta" >}}
+{{< feature-state for_k8s_version="v1.26" state="GA" >}}
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/endpoint-slices.md:101]:
```suggestion
{{< feature-state for_k8s_version="v1.26" state="stable" >}}
```

File: content/en/docs/concepts/services-networking/endpoint-slices.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ for terminating pods independent of the existing semantics for `ready`.
 
 #### Terminating
 
-{{< feature-state for_k8s_version="v1.22" state="beta" >}}
+{{< feature-state for_k8s_version="v1.26" state="GA" >}}
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/endpoint-slices.md:120]:
```suggestion
{{< feature-state for_k8s_version="v1.26" state="stable" >}}
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -262,6 +260,9 @@ For a reference to old feature gates that are removed, please refer to
 | `EfficientWatchResumption` | `false` | Alpha | 1.20 | 1.20 |
 | `EfficientWatchResumption` | `true` | Beta | 1.21 | 1.23 |
 | `EfficientWatchResumption` | `true` | GA | 1.24 | - |
+| `EndpointSliceTerminatingCondition` | `false` | Alpha | 1.20 | 1.21 |
+| `EndpointSliceTerminatingCondition` | `true` | Beta | 1.22 | 1.26 |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:264]:
```suggestion
| `EndpointSliceTerminatingCondition` | `true` | Beta | 1.22 | 1.25 |
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 4]
```diff
@@ -606,7 +607,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   filesystem walk for better performance and accuracy.
 - `LogarithmicScaleDown`: Enable semi-random selection of pods to evict on controller scaledown
   based on logarithmic bucketing of pod timestamps.
-- `MatchLabelKeysInPodTopologySpread`: Enable the `matchLabelKeys` field for 
+- `MatchLabelKeysInPodTopologySpread`: Enable the `matchLabelKeys` field for
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:610]: Ideally, don't let your editor touch this line.

**Review by sftim — COMMENTED** [review]
Hi. These changes should fix the build.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 4 above)
Comment (andrewsykim) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:610]: Sorry, my editor is configured to automatically format any file on save :upside_down_face:

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 5]
```diff
@@ -603,7 +604,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
```
Comment (andrewsykim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:607]:
```suggestion
- `MatchLabelKeysInPodTopologySpread`: Enable the `matchLabelKeys` field for 
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 6]
```diff
@@ -646,7 +647,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
```
Comment (andrewsykim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:650]:
```suggestion
- `MatchLabelKeysInPodTopologySpread`: Enable the `matchLabelKeys` field for 
```

**Comment by leonardpahlke** [comment]
Awesome, lets get this in ASAP for 1.26

**Review by sftim — COMMENTED** [review]
Are these condition names written  camel case (`camelCaseExample` aka `lowerCamelCaseExample`), Pascal case (`PascalCaseExample`), initial caps (`Initialcapsexample`) or something else?

For GA, we should get details like this right.

**Comment by sftim** [comment]
This looks ready for tech review.

**Comment by robscott** [comment]
Thanks @andrewsykim!

/lgtm

**Comment by sftim** [comment]
Hi. This now needs to target _main_

/lgtm cancel

**Pull request closed without merging**